lib/rex/compat.rb

```diff
@@ -35,10 +35,10 @@ module Compat
 @@loaded_tempfile  = false
 @@loaded_fileutils = false
 
-
 def self.is_windows
   return @@is_windows if @@is_windows
```

Comment:
Unrelated; It looks like there's a bug here - if `@@is_windows` is true, this method will be faster on future lookups as it returns early with the guard clause. If it's false, we'll always re-evaluate `@@is_windows` as false.

The class should probably initialize its internal state as `nil`, and use `return @@is_windows unless @@is_windows.nil?` instead